plugins/usbdmx/UsbDmxPlugin.cpp

```diff
@@ -107,6 +108,15 @@ bool UsbDmxPlugin::SetDefaultPreferences() {
 
   return true;
 }
+
+void UsbDmxPlugin::ConflictsWith(
+  std::set<ola_plugin_id>* conflicting_plugins) const {
+  if (EuroliteProFactory::IsEuroliteMk2Enabled(m_preferences)) {
+    conflicting_plugins->insert(OLA_PLUGIN_USBPRO);
+    conflicting_plugins->insert(OLA_PLUGIN_FTDIDMX);
```

Comment:
Stageprofi should probably be added to this list too I think.

Reply:
I've added it, and added it to the readme.